### PR TITLE
AUTH-WEB - Remove affiliation invitation from main branch.

### PR DESCRIPTION
**Pull request by seeker25**
By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of the sbc-auth license (Apache 2.0).

### CODE REVIEW

File: auth-web/src/components/auth/manage-business/AffiliatedEntityTable.vue [hunk 1]
```diff
@@ -502,7 +502,7 @@ export default defineComponent({
     }
 
     const getRequestForAuthorizationStatusText = (affiliationInviteInfos: AffiliationInviteInfo[]) => {
-      if (isCurrentOrganization(affiliationInviteInfos[0].toOrg.id)) {
+      if (isCurrentOrganization(affiliationInviteInfos[0].toOrg?.id)) {
```
Comment (ochiu) [inline auth-web/src/components/auth/manage-business/AffiliatedEntityTable.vue:505]: What if it is an empty affiliation invitation list? is this checked elsewhere before invoked?

**Review by hfekete — APPROVED** [review]

File: auth-web/src/components/auth/manage-business/AffiliatedEntityTable.vue (hunk 1 above)
Comment (seeker25) [reply auth-web/src/components/auth/manage-business/AffiliatedEntityTable.vue:505]:
```
          <span
            v-if="!!item.affiliationInvites"
            id="affiliationInvitesStatus"
          >
            <p style="font-size: 12px">
              <v-icon
                x-small
                color="primary"
              >mdi-account-cog</v-icon>
              <span v-html="getRequestForAuthorizationStatusText(item.affiliationInvites)" />
            </p>
          </span>
        </template>
        ```

Comment (seeker25) [reply auth-web/src/components/auth/manage-business/AffiliatedEntityTable.vue:505]: Looks like it's checked there

**Review by ochiu — APPROVED** [review]

**Review by ochiu — APPROVED** [review]

**Comment by seeker25** [comment]
This is going to cause some conflicts when back merging from main -> feature. But we'll take the feature branch changes over main.

File: auth-web/src/models/affiliation.ts [hunk 2]
```diff
@@ -41,7 +41,7 @@ export class AffiliationInviteInfo {
   }
 
   static isToOrg (affiliationInviteInfo: AffiliationInviteInfo, toOrgId: number): boolean {
-    return affiliationInviteInfo.toOrg.id === toOrgId
+    return affiliationInviteInfo.toOrg?.id === toOrgId
```
Comment (seeker25) [inline auth-web/src/models/affiliation.ts:44]: Probably could have removed this as well, just going to leave for now.

**Comment by bcregistry-sre** [comment]
Temporary Url for review: https://bcregistry-account-dev--pr-2499-hs20yw69.web.app

**Pull request merged**